classes/Cart.php

```diff
@@ -1952,7 +1952,8 @@ public function getOrderTotal(
         // CART CALCULATION
         $cartRules = array();
         if (in_array($type, [Cart::BOTH, Cart::BOTH_WITHOUT_SHIPPING, Cart::ONLY_DISCOUNTS])) {
-            $cartRules = $this->getCartRules();
+            $cartRules = $this->getTotalCalculationCartRules($type, $type == Cart::BOTH);
+            //$cartRules = $this->getCartRules();
```

Comment:
@tomlev: * Comments code in case we need it later *

Git:
![Am I a joke to you](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSVAh70cebbFth27lWIP2CchcxQpn9XYCy-6GUDOj5yVwdBlsep)

Reply:
commented in case of english people wishing to change their mind as they usually do ^^